Specs/Scene/ResourceCacheSpec.js

```diff
@@ -411,9 +377,7 @@ describe(
       });
 
       expect(function () {
-        ResourceCache.unload({
-          resourceLoader: resourceLoader,
-        });
+        ResourceCache.unload(resourceLoader);
```

Comment:
Turns out that error can't actually happen because the resource would have already been removed from the cache and the previous error would get hit. I removed the check and updated the name of the test.